docs/application/vstools/install.md

```diff
@@ -1,170 +1,102 @@
 
-   ![Programs and Features](media/cs_prerequisite01-250x401.png)
+# Visual Studio Tools for Tizen 
 
-  2. Select **Turn Windows features on or off**.
-  3. Disable **Hyper-V** and click **OK**.
-
-   ![Disable Hyper-V](media/cs_prerequisite-disable-hiperv.png)
-
-  4. Reboot the computer.
+To use the Tizen SDK tools in Visual Studio, you must install:
 
+1.	[Visual Studio Tools for Tizen Extension](#installing-visual-studio-tools-for-tizen-extension )
+2.	[Tizen Baseline SDK](#installing-tizen-baseline-sdk)
 
 <a name="install"></a>
-## Visual Studio Tools for Tizen Installation
-
-To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the baseline SDK.
-
-> **Note**  
-> If the Tizen Studio is already installed in your computer, simply [set the Tizen baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
-
-
-### Installing the Extension
-
-The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can install extensions from the Visual Studio Marketplace in the Visual Studio IDE.
-
-1. In the Visual Studio IDE menu, go to **Tools &gt; Extensions and Updates**.
-2. In the Visual Studio Marketplace, search for **Tizen**.
-
-   ![Marketplace](media/howtoinstall-extensionwindows.png)
-
-3. Click **Download** and close the Visual Studio IDE.
-
-   The installation starts.
-
-Alternatively, you can download the Visual Studio Tools for Tizen extension from the [Visual Studio Marketplace Web site](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
+## Install Visual Studio Tools for Tizen Extension 
 
-### Installing the Tizen Baseline SDK
+The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can install the extensions from the Visual Studio Marketplace in the Visual Studio IDE.
 
-After installing the Visual Studio Tools for Tizen extension, you must set up the Tizen baseline SDK:
+1.	In the Visual Studio IDE menu, go to **Tools > Extensions and Updates**.
+2.	In the Visual Studio Marketplace, search for **Tizen**. The **Visual Studio Tools for Tizen** option appears. 
+3.	Click **Download** and close the Visual Studio IDE. \
+The installation begins.
 
-1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-2. Select **Install new Tizen SDK**.
+![Visual Studio Tools for Tizen](media\v1-install-tools.gif)
 
-   ![Select new installation](media/howtoinstall-installwizard1.png)
+Alternatively, you can download the Visual Studio Tools for Tizen extension from [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
 
-3. Read the license document and click **I Agree**.
+## Install Tizen Baseline SDK
 
```

Comment:
Information Required: Could you provide some more details to give an overview. Why do we need to install?

Reply:
Baseline SDK contains Tizen-specific libraries and tools such as profilers and debugger.